homeassistant/components/mystrom/switch.py

```diff
@@ -71,7 +91,7 @@ def is_on(self):
     @property
     def unique_id(self):
         """Return a unique ID."""
-        return self.plug._mac  # pylint: disable=protected-access
+        return format_mac(self.plug.mac)
```

Comment:
Same as light => this should not be formatted.

Reply:
removed the fromat